torch/nn/functional.py

```diff
@@ -1719,15 +1719,23 @@ def kl_div(input, target, size_average=None, reduce=None, reduction='mean'):
             on :attr:`size_average`. When :attr:`reduce` is ``False``, returns a loss per
             batch element instead and ignores :attr:`size_average`. Default: ``True``
         reduction (string, optional): Specifies the reduction to apply to the output:
-            'none' | 'mean' | 'sum'. 'none': no reduction will be applied,
-            'mean': the sum of the output will be divided by the number of
-            elements in the output, 'sum': the output will be summed. Note: :attr:`size_average`
-            and :attr:`reduce` are in the process of being deprecated, and in the meantime,
-            specifying either of those two args will override :attr:`reduction`. Default: 'mean'
+            'none' | 'batchmean' | 'sum' | 'mean'. 'none': no reduction will be applied,
+            'batchmean': the sum of the output will be divided by the number of
+            batches in the output, 'sum': the output will be summed, 'mean': the output will be
```

Comment:
"the number of batches" -> "the batch size" or "the number of elements in the input batch"